src/NLog/Layouts/XmlElementBase.cs

```diff
@@ -337,7 +336,7 @@ private void RenderXmlFormattedMessage(LogEventInfo logEvent, StringBuilder sb)
                         RenderStartElement(sb, ElementNameInternal);
                     }
                     int beforeValueLength = sb.Length;
-                    ElementValueInternal.RenderAppendBuilder(logEvent, sb);
+                    _elementValueWrapper.RenderAppendBuilder(logEvent, sb);
```

Comment:
@304NotModified This is the bug. Rendered using the Inner-Layout instead of XmlEncodeLayoutRendererWrapper